presto-main/src/test/java/io/prestosql/operator/scalar/TestStringFunctions.java

```diff
@@ -1046,4 +1046,27 @@ public void testCharConcat()
 
         assertFunction("concat(cast(null as char(1)), cast(' ' as char(1)))", createCharType(2), null);
     }
+
+    @Test
+    public void testTranslate()
+    {
+        //assertFunction("translate('abcd', '', '')", VARCHAR, "abcd");
```

Comment:
Does this not work? This seems like a good test case to have, either working or as an explicitly disallowed input.

Reply:
No, accidentally left commented out.  Fixed.